src/components/carousel-with-image/CarouselWithImage.styles.ts

```diff
@@ -17,7 +17,8 @@ export const styles = {
     minHeight: '144px',
     backgroundColor: 'primary.800',
     borderRadius: '6px',
-    p: '16px'
+    p: '16px',
+    m: '0 1.5vw'
```

Comment:
I guess, it should have a static value, which does not depend on window size

Reply:
> I guess, it should have a static value, which does not depend on window size

As i understand, we have responsive svg above description block with "fake" white spaces (white spaces inside svg). 

If we use static value we got next result where gaps between blocks is not the same as between svgs. 
![static](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/d459f799-f408-4808-83b2-f90234a788cc)

But if we use vw, this resolve the problem
![adaptive](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/f69cdfc4-dcd2-4b76-bf5c-69e5055c4612)